bigbluebutton-html5/imports/ui/components/error-screen/component.jsx

```diff
@@ -80,40 +78,59 @@ const propTypes = {
     PropTypes.string,
     PropTypes.number,
   ]),
+  error: PropTypes.object,
+  errorInfo: PropTypes.object,
 };
 
 const defaultProps = {
   code: '500',
   callback: () => {},
   endedReason: null,
+  error: {},
+  errorInfo: {},
 };
 
 class ErrorScreen extends PureComponent {
   componentDidMount() {
     const { code, callback, endedReason } = this.props;
-    const log = code === '403' ? 'warn' : 'error';
-    AudioManager.exitAudio();
+    // stop audio
+    document.querySelector('audio').pause();
+    navigator
+      .mediaDevices
+      .getUserMedia({ audio: true, video: true })
+      .then((m) => m.getTracks().forEach((t) => t.stop()));
     callback(endedReason, () => Meteor.disconnect());
-    logger[log]({ logCode: 'startup_client_usercouldnotlogin_error' }, `User could not log in HTML5, hit ${code}`);
+    console.error({ logCode: 'startup_client_usercouldnotlogin_error' }, `User could not log in HTML5, hit ${code}`);
```

Comment:
logger could be POSTed to the server. console.error is client only. Any chance we can keep logger?

Reply:
The logger is heavely dependent of the settings that are loaded after this component be loaded